src/documentation/ntfs_to_gtfs_specs.md

```diff
@@ -1,6 +1,6 @@
 # NTFS to GTFS conversion
 ## Introduction
-This document describes how a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) is transformed into a [GTFS feed ](https://developers.google.com/transit/gtfs/reference/) in Navitia Transit Model.
+This document describes how a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) is transformed into a [GTFS] feed in Navitia Transit Model.
```

Comment:
See similar comment in `src/documentation/gtfs_to_ntfs_specs.md`.

Reply:
Resolving it to group discussion in the other comment ;)